packages/notifications/src/Notification/Notification.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useRef } from 'react';
+import { Alert, TextContent, Text, TextVariants, AlertActionCloseButton, AlertVariant } from '@patternfly/react-core';
+import { CloseIcon } from '@patternfly/react-icons';
+import './notification.scss';
+
+export interface NotificationProps {
```

Comment:
This will be a lengthy discussion I can say it right now :smile: 

Do we want to use `interface` for this one? Instead of `type`? I went trough couple of articles and the decision is on us, I just want to understand if we know what is the reasoning.

Basically I would use interface for props that will be extended by apps, meaning someone were to import `NotificationProps` add their own props and then use that interface. Which is mostly not the case. Morever this can be achieved with types as well. If we were to write classes I would be completely fine with writing interface for that case.

Reply:
I used an interface here because I use this type but changed the id attribute for the redux functions as they do not require it.

My POV at the interface/type discussions is that components prop is a component interface, therefor I used interface. But honestly, I don't really care. It does the same thing in the end.

Reply:
I went trough a couple of articles and the most recent ones mentioned that `interfaces` should be used if you expect the consumer to export it. I'd like to come with same approach for all typings honestly, so let's either go with all interfaces or prefer `type` (which is better when typing funcitons) and only rarerly use `interface` (I would prefer type as it looks like it's the easier and more straight forward case).

Reply:
As I said. I don't really mind either way. This specific has to be an interface for the reasons above. Once we verify the build works and the package is usable, We can create some sort of manifesto or w/e it called and come up will all the TS conventions we want to use in the future.